libs/internal/test-util/src/lib/logs/log-creators.ts

```diff
@@ -1,20 +1,20 @@
 import { TestBed } from '@angular/core/testing';
 import { LumberjackLogLevel, LumberjackLevel, LumberjackTimeService } from '@ngworker/lumberjack';
 
-const createLog = (level: LumberjackLogLevel, message = '', context = 'Test') => ({
-  context,
+const createLog = (level: LumberjackLogLevel, message = '', scope = 'Test') => ({
+  scope,
   createdAt: TestBed.inject(LumberjackTimeService).getUnixEpochTicks(),
   level,
   message,
 });
-export const createCriticalLog = (message?: string, context?: string) =>
-  createLog(LumberjackLevel.Critical, message, context);
-export const createDebugLog = (message?: string, context?: string) =>
-  createLog(LumberjackLevel.Debug, message, context);
-export const createErrorLog = (message?: string, context?: string) =>
-  createLog(LumberjackLevel.Error, message, context);
-export const createInfoLog = (message?: string, context?: string) => createLog(LumberjackLevel.Info, message, context);
-export const createTraceLog = (message?: string, context?: string) =>
-  createLog(LumberjackLevel.Trace, message, context);
-export const createWarningLog = (message?: string, context?: string) =>
-  createLog(LumberjackLevel.Warning, message, context);
+export const createCriticalLog = (message?: string, scope?: string) =>
+  createLog(LumberjackLevel.Critical, message, scope);
+export const createDebugLog = (message?: string, scope?: string) =>
+  createLog(LumberjackLevel.Debug, message, scope);
+export const createErrorLog = (message?: string, scope?: string) =>
+  createLog(LumberjackLevel.Error, message, scope);
+export const createInfoLog = (message?: string, scope?: string) => createLog(LumberjackLevel.Info, message, scope);
+export const createTraceLog = (message?: string, scope?: string) =>
+  createLog(LumberjackLevel.Trace, message, scope);
+export const createWarningLog = (message?: string, scope?: string) =>
+  createLog(LumberjackLevel.Warning, message, scope);
```

Comment:
We should see if it's appropriate and useful to replace this with `LumberjackLogBuilder`.